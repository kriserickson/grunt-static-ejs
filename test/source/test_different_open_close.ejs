<!DOCTYPE html>
<html>
<head>
    <title>Test</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    [% if (development) { %]
        <link href="css/style1.css" rel="stylesheet">
        <link href="css/style2.css" rel="stylesheet">
        <link href="css/style3.css" rel="stylesheet">
        <script src="js/js1.js"></script>
        <script src="js/js2.js"></script>
        <script src="js/js3.js"></script>
    [% } else { %]
        <link href="css/merged.css" rel="stylesheet">
        <script src="js/merged.js"></script>
    [% } %]

</head>
<body>

<div data-role="page" id="loading">
    <content>
        [%= content %]
    </content>
</div>

<script type="text/template" id="noteTemplate">
    <div><%= name %></div>
    <h5>Note Created On : <%= date %></h5>
    <h5><%= text %></h5>
</script>

</body>
</html>



