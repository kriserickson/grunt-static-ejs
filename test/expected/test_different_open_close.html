<!DOCTYPE html>
<html>
<head>
    <title>Test</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
        <link href="css/merged.css" rel="stylesheet">
        <script src="js/merged.js"></script>
    

</head>
<body>

<div data-role="page" id="loading">
    <content>
        Test Content
    </content>
</div>

<script type="text/template" id="noteTemplate">
    <div><%= name %></div>
    <h5>Note Created On : <%= date %></h5>
    <h5><%= text %></h5>
</script>

</body>
</html>



